gson/src/test/java/com/google/gson/stream/JsonReaderTest.java

```diff
@@ -778,10 +782,11 @@ public void testPeekMuchLargerThanLongMinValue() throws IOException {
       fail();
     } catch (NumberFormatException expected) {
     }
-    assertThat(reader.nextDouble()).isEqualTo(-92233720368547758080d);
+    assertThat(reader.nextDouble()).isEqualTo(d);
   }
 
   @Test
+  @SuppressWarnings("UnicodeEscape")
   public void testQuotedNumberWithEscape() throws IOException {
     JsonReader reader = new JsonReader(reader("[\"12\u00334\"]"));
```

Comment:
This is actually a bug with the test; the Java Compiler currently handles the escape, so Gson does not actually process it. The escape should use `\\`.
```suggestion
  public void testQuotedNumberWithEscape() throws IOException {
    JsonReader reader = new JsonReader(reader("[\"12\\u00334\"]"));
```